main.py

```diff
@@ -2,6 +2,49 @@
 
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
-    pass
+    genres = ("Western", "Action", "Dramma")
+    actors = (
+        "George Klooney",
+        "Kianu Reaves",
+        "Scarlett Keegan",
+        "Will Smith",
+        "Jaden Smith",
+        "Scarlett Johansson"
+    )
```

Comment:
why not create a list of tuples? you will not need to use split later

Reply:
Emm... cause I confused good and bad practice examples in checklist. And it was late so I already didn't think by myself😅